test/runtests.jl

```diff
@@ -176,7 +176,7 @@ end
 
 @testset "Gamma IdentityLink" begin
     gm10 = fit(GeneralizedLinearModel, lot1 ~ 1 + u, clotting, Gamma(), IdentityLink(),
-        convTol=1e-8)
```

Comment:
Actually this is in response to the review above that I had de-indented most of runtests.jl